webdriver/tests/bidi/script/evaluate/exception_details_await_promise.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from webdriver.bidi.modules.script import ContextTarget, ScriptEvaluateResultException
+
+from ... import any_int, any_string, recursive_compare
+from .. import any_stack_trace
+from . import PRIMITIVE_AND_REMOTE_VALUES
+
+
+@pytest.mark.asyncio
+@PRIMITIVE_AND_REMOTE_VALUES
+async def test_exception_details_await_promise(bidi_session, top_context,
+                                               expression, expected):
```

Comment:
Why not the following which we use in a lot of other places?

```suggestion
async def test_exception_details_await_promise(
    bidi_session, top_context, expression, expected
):
```